### PR TITLE
feat: add unsupported feature page

### PR DESCRIPTION
**Pull request by AnjaMis**
This PR adds a "Feature not supported on this device" page mentioned in [this](https://github.com/okp4/dataverse-portal/issues/317) issue. 

### Screenshots

<details>
<summary> Tablet </summary>

localhost_5173_

</details> 

<details>
<summary> Mobile </summary>

localhost_5173_(iPhone 6_7_8)

</details>

### CODE REVIEW

File: src/ui/page/share/share.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { useCallback } from 'react'
 import { useNavigate } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
 import { Card } from '@/ui/component/card/card'
+import { useBreakpoint } from '@/ui/hook/useBreakpoint'
+import { UnsupportedFeature } from '../../view/unsupportedFeature/unsupportedFeature'
```
Comment (lolottetheclash) [inline src/ui/page/share/share.tsx:7]: you can use alias here 😉

File: src/ui/view/unsupportedFeature/unsupportedFeature.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <div className="okp4-dataverse-portal-unsupported-feature-image" />
+        <div className="okp4-dataverse-portal-unsupported-feature-text-container">
+          <div className="okp4-dataverse-portal-unsupported-feature-titles">
+            <h1 className="okp4-dataverse-portal-unsupported-feature-title">
+              {t('unsupportedFeature.sorry')}
+            </h1>
+            {isTablet ? (
+              <h1>{t('unsupportedFeature.titleTablet')}</h1>
+            ) : (
+              <h1>{t('unsupportedFeature.titleMobile')}</h1>
```
Comment (lolottetheclash) [inline src/ui/view/unsupportedFeature/unsupportedFeature.tsx:30]: here you can shorten the code with interpolation

**Review by lolottetheclash — CHANGES_REQUESTED** [review]
This is nice and working well, good job!
Minor remarks and it will be good for me!

File: src/ui/page/share/share.tsx (hunk 1 above)
Comment (AnjaMis) [reply src/ui/page/share/share.tsx]: Completely missed it, thanks! :)

File: src/ui/view/unsupportedFeature/unsupportedFeature.tsx (hunk 2 above)
Comment (AnjaMis) [reply src/ui/view/unsupportedFeature/unsupportedFeature.tsx]: Ah didn't think about that, should be fixed now!

**Review by ccamel — APPROVED** [review]
LGTM! :+1:

File: src/ui/view/unsupportedFeature/unsupportedFeature.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import { Button } from '@/ui/component/button/button'
+import { useBreakpoint } from '@/ui/hook/useBreakpoint'
+import './unsupportedFeature.scss'
+import '@/ui/view/unsupportedFeature/i18n/index'
+
+export const UnsupportedFeature: FC = () => {
+  const { t } = useTranslation('unsupportedFeature')
+
+  const navigate = useNavigate()
+  const handleReturnToHome = useCallback(() => navigate('/'), [navigate])
```
Comment (fredericvilcot) [inline src/ui/view/unsupportedFeature/unsupportedFeature.tsx:14]: Would be better here to use the `routes.home` enum key :)

**Review by fredericvilcot — CHANGES_REQUESTED** [review]
Great, just a minor remark 😌

**Review by 0xLodz — APPROVED** [review]
LGTM!

**Review by lolottetheclash — APPROVED** [review]
👍

**Review by fredericvilcot — APPROVED** [review]
👌

**Pull request merged**